docker-compose.yml

```
@@ -1,58 +1,55 @@
 version: '3'
 
+x-restart-policy: &default_restart_policy
+  condition: on-failure
+  delay: 3s
+  max_attempts: 5
+  window: 60s
 services:
   redis:
     deploy:
       replicas: 1
       placement:
         constraints:
-          - node.labels.role == opencti-core
-      restart_policy: &default_restart_policy
-        condition: on-failure
-        delay: 3s
-        max_attempts: 5
-        window: 60s
-    image: redis:7.2.4
-
+        - node.labels.role == opencti-core
+    image: redis:7.4.2
+    restart_policy: *default_restart_policy
     volumes:
       - redisdata:/data
-
+      
   nginx:
     image: codeforafrica/cfa-opencti:0.0.6
     deploy:
       replicas: 1
       placement:
         constraints:
-          - node.labels.role == manager
-      restart_policy: *default_restart_policy
+        - node.labels.role == manager
     ports:
       - "80:80"
     depends_on:
       - opencti
 
   rabbitmq:
-    image: rabbitmq:3.13-management
-    deploy:
-      replicas: 1
-      placement:
-        constraints:
-          - node.labels.role == opencti-core
-      restart_policy: *default_restart_policy
+    image: rabbitmq:4.0-management
     environment:
       - RABBITMQ_DEFAULT_USER=${RABBITMQ_DEFAULT_USER}
       - RABBITMQ_DEFAULT_PASS=${RABBITMQ_DEFAULT_PASS}
       - RABBITMQ_NODENAME=rabbit01@localhost
     volumes:
       - amqpdata:/var/lib/rabbitmq
+    restart_policy: *default_restart_policy
+    deploy:
+      placement:
+        constraints:
+          - node.labels.role == opencti-core
```

Isn't `restart_policy` supposed to be part of `deploy`?
```suggestion
    deploy:
      placement:
        constraints:
          - node.labels.role == opencti-core
      restart_policy: *default_restart_policy
```